src/DoctrineORMModule/Module.php

```diff
@@ -139,7 +138,14 @@ public function initializeConsole(EventInterface $event)
         $entityManager = $serviceLocator->get('doctrine.entitymanager.orm_default');
         $helperSet     = $cli->getHelperSet();
 
-        $helperSet->set(new QuestionHelper(), 'question');
+        if (class_exists('Symfony\Component\Console\Helper\QuestionHelper')) {
+            $helperSet->set(new \Symfony\Component\Console\Helper\QuestionHelper(), 'question');
+        }
+
+        if (class_exists('Symfony\Component\Console\Helper\DialogHelper')) {
+            $helperSet->set(new \Symfony\Component\Console\Helper\DialogHelper(), 'dialog');
```

Comment:
I set both to keep backwards compatibility for [symfony/console ~2.5](https://github.com/symfony/Console/tree/2.5/Helper) where both are available.